src/views/Team/Outlet.vue

```diff
@@ -101,235 +193,292 @@
+import Parse from "parse";
+import { takePicture, Photo } from "@/utils/camera";
+import { Model } from "@/utils/model";
+import { absoluteUrl } from "@/utils/url";
 
 const DEFAULT_STYLES = {
-  "background": "transparent",
-  "backgroundImage": "linear-gradient(to right, var(--ion-color-secondary) 0%, var(--ion-color-primary) 100%)"
-}
+  background: "transparent",
+  backgroundImage:
+    "linear-gradient(to right, var(--ion-color-secondary) 0%, var(--ion-color-primary) 100%)",
+};
 
 export default defineComponent({
-  name: 'Team',
-  data(){
+  name: "Team",
+  data() {
     return {
-      showQr: false,
       loading: true,
-    }
+      state: "about",
+      subteam: "",
+    };
   },
   setup() {
     const store = useStore();
-
     return {
       store,
-      chatbubbles, logoWhatsapp, uploadIcon: cloudUploadOutline,
-      editIcon, qrCodeIcon, imageIcon, trashIcon, addIcon,
-    }
+      chatbubbles,
+      logoWhatsapp,
+      uploadIcon: cloudUploadOutline,
+      editIcon,
+      qrCodeIcon,
+      imageIcon,
+      trashIcon,
+      addIcon,
+    };
   },
   watch: {
     // call again the method if the route changes
-    '$route': 'fetchData'
+    $route: "fetchData",
   },
   computed: {
     team(): Model {
       const slug: any = this.$route.params.teamSlug;
-      return this.store.getters.objectsMap[this.store.getters.teamsBySlug[slug]]
+      console.log("Slug abaixo");
+      console.log(
+        this.store.getters.objectsMap[this.store.getters.teamsBySlug[slug]]
+      );
+      return this.store.getters.objectsMap[
+        this.store.getters.teamsBySlug[slug]
+      ];
     },
     feed(): Model[] {
       if (!this.team) {
-        return []
+        return [];
       }
       const teamId = this.team.objectId;
-      return this.store.state.teams.news[teamId]
-        .map((id: string) => this.store.getters["objectsMap"][id])
+      return this.store.state.teams.news[teamId].map(
+        (id: string) => this.store.getters["objectsMap"][id]
+      );
     },
     subteams(): Model[] {
       if (!this.team) {
-        return []
+        return [];
       }
       const teamId = this.team.objectId;
-      return (this.store.state.teams.subteams[teamId] || [])
-        .map((id: string) => this.store.getters["objectsMap"][id])
+      return (this.store.state.teams.subteams[teamId] || []).map(
+        (id: string) => this.store.getters["objectsMap"][id]
+      );
     },
     info(): string {
-      return this.team.info || ""
+      return this.team.info || "";
     },
     socialLinks(): any[] {
-      return this.team.socialLinks || []
+      return this.team.socialLinks || [];
     },
     footerLinks(): any[] {
-      return this.team.footerLinks || []
+      return this.team.footerLinks || [];
+    },
+    subOf(): any {
+      console.log(this.team.subOf);
+      return this.team.subOf.name || "";
+    },
+    teamhahay(): any {
+      console.log("haylou");
+      return "olamundo";
```

Comment:
what's this?

Reply:
Sorry about that, it was when testing at first and did not delete (Removed at this point)